firstvoices/backend/views/sites_views.py

```diff
@@ -133,6 +135,18 @@ def get_serializer_context(self):
             context["site"] = self.get_object()
         return context
 
+    def handle_exception(self, exc):
```

Comment:
Is this necessary? I would expect DRF to turn NotAuthenticated into a 401 automatically